eng/Version.Details.xml

```diff
@@ -1,139 +1,139 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="VS.Tools.Net.Core.SDK.Resolver" Version="9.0.100-preview.6.24328.19">
+    <Dependency Name="VS.Tools.Net.Core.SDK.Resolver" Version="9.0.100-preview.7.24360.5">
       <Uri>https://github.com/dotnet/sdk</Uri>
-      <Sha>ef4c24166691977558e5312758df4313ab310dc0</Sha>
+      <Sha>9a028e12b90e6a583b09ccb3008fdfaf85761f19</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.6.24327.7" CoherentParentDependency="VS.Tools.Net.Core.SDK.Resolver">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="9.0.0-preview.7.24357.2" CoherentParentDependency="VS.Tools.Net.Core.SDK.Resolver">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>573461dc897731c11d6166574b77a8c3aabba220</Sha>
+      <Sha>4e278fe17f69ea31fbdcbab74ac47ec6fa84914b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.99.0-preview.6.340">
+    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="34.99.0-preview.7.346">
       <Uri>https://github.com/dotnet/android</Uri>
-      <Sha>ef5fd216aa83867f13b8abdc0986d9373151a14e</Sha>
+      <Sha>06bb1dc6a292ef5618a3bb6ecca3ca869253ff2e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.MacCatalyst.Sdk.net9.0_17.2" Version="17.2.9714-net9-p6">
+    <Dependency Name="Microsoft.MacCatalyst.Sdk.net9.0_17.2" Version="17.2.9634-net9-p6">
```

Comment:
This looks like a downgrade, where did you get this version from?

Reply:
Seems thats the last one on the channel. But i reverted 

```
PS C:\repos\dotnet\maui> darc update-dependencies --channel ".NET 9.0.1xx SDK"
Looking up latest build of https://github.com/dotnet/sdk on .NET 9.0.1xx SDK
Looking up latest build of https://github.com/dotnet/android on .NET 9.0.1xx SDK
Looking up latest build of https://github.com/xamarin/xamarin-macios on .NET 9.0.1xx SDK
Looking up latest build of https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal on .NET 9.0.1xx SDK
Looking up latest build of https://github.com/dotnet/aspnetcore on .NET 9.0.1xx SDK
Looking up latest build of https://github.com/dotnet/templating on .NET 9.0.1xx SDK
Looking up latest build of https://github.com/dotnet/xharness on .NET 9.0.1xx SDK
Looking up latest build of https://github.com/dotnet/arcade on .NET 9.0.1xx SDK
Updating 'Microsoft.MacCatalyst.Sdk.net9.0_17.2': '17.2.9714-net9-p6' => '17.2.9634-net9-p6' (from build '20240528.4' of 'https://github.com/xamarin/xamarin-macios')
Checking for coherency updates...
Local dependencies updated from channel '.NET 9.0.1xx SDK'.
```